hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestFSNamesystemMBean.java

```diff
@@ -225,4 +229,44 @@ public void testFsEditLogMetrics() throws Exception {
       }
     }
   }
+
+  /**
+   * Test metrics associated with reconstructionQueuesInitProgress.
+   */
+  @Test
+  public void testReconstructionQueuesInitProgressMetrics() throws Exception {
+    Configuration conf = new Configuration();
+    try (MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf).numDataNodes(1).build()) {
```

Comment:
The `numDataNodes(1)` is same as the default value, so it should be removed if equal to default.

Reply:
yeah, i will update later.